### PR TITLE
Support k-many modalities and techniques

### PR DESCRIPTION
**Pull request by Dtphelan1**
One more small piece of feedback from Michelle and Anthony - there can be 0...* many modality-and-technique extension instances on RTTD resources. This MR takes the list of k-many instances and joins them in a comma-space-separated list of display strings. It also creates a few generic functions for getting Techniques, Modalities, Procedure Intents, and Body Sites.

image

### CODE REVIEW

**Review by dmendelowitz — COMMENTED** [review]
Based on what Michelle said (modalities and techniques being pairs, 1st modalities matching to 1st technique), do you think it would be good to line them up visually somehow? It could be kind of hard to parse with the text wrapping. Might not be necessary, maybe we show Michelle the current state of things and see what she thinks?

**Comment by Dtphelan1** [comment]
Great point Dylan. I showed Michelle how the initial attempt looked, and she recommended joining the values with a newline instead of with a comma for better readability. I showed her the result and she's confirmed that this new approach is sufficient for now!

**Review by dmendelowitz — APPROVED** [review]
